backend/src/utils.ts

```diff
@@ -8,13 +8,24 @@ type NodeType = {
 }
 
 const typeToElStrMap = {
-  'block-quote': 'blockquote',
+  'heading-one': 'h1',
   'heading-two': 'h2',
+  'block-quote': 'blockquote',
   'bulleted-list': 'ul',
   'numbered-list': 'ol',
   paragraph: 'p',
 }
 
+const nonBodyTypes = new Set(['heading-one', 'heading-two', 'block-quote'])
+
+const breakCharacters = new Set([
+  ' ', // Good ole ASCII space
+  '.',
+  ';',
+  '　', // Full width space (Japanese, Chinese?)
+  '。', // Ideographic full stop
+])
+
```

Comment:
Very noiceeeee, thanks for adding this in!! Since I type almost every day in Chinese or Japanese, I deal with the full-width spaces and 。ideographic full stop all the time. Glad we have this covered!